sphinx/tutorial/cylc/scheduling/datetime-cycling.rst

```diff
@@ -183,21 +183,23 @@ Recurrence Formats
    When using durations, beware that a change in the initial cycle point
    might produce different results for the recurrences.
 
-   For example if you set the initial cycle point to ``2000-01-01T00`` the
-   recurrence ``P1D`` would yield:
+   .. ifnotslides::
 
-   ``2000-01-01T00``, ``2000-01-02T00``, ``2000-01-03T00``, ...
+      For example if you set the initial cycle point to ``2000-01-01T00`` the
+      recurrence ``P1D`` would yield:
 
-   If, however, the initial cycle point was changed from midnight to midday
-   (``2000-01-01T12``), the same recurrence would instead yield:
+      ``2000-01-01T00``, ``2000-01-02T00``, ``2000-01-03T00``, ...
 
-   ``2000-01-01T12``, ``2000-01-02T12``, ``2000-01-03T12``, ...
+      If, however, the initial cycle point was changed from midnight to midday
+      (``2000-01-01T12``), the same recurrence would instead yield:
 
-   This can easily be adjusted. Both of the following recurrences induce a
-   start on the first midnight *after* the initial cycle point.
+      ``2000-01-01T12``, ``2000-01-02T12``, ``2000-01-03T12``, ...
 
-   * ``T00/P1D``
-   * ``T00``
+      This can easily be adjusted. Both of the following recurrences induce a
+      start on the first midnight *after* the initial cycle point.
+
+      * ``T00/P1D``
+      * ``T00``
```

Comment:
Perhaps add a short final sentence along the lines of 'This is why *explicit* recurrences such as ``T00`` are preferred to / safer than *implicit* ones such as ``P1D``' to hit the point home.

Reply:
I agree, this section is kinda strange so I've re-written it, see what you think. I haven't expressed a solid preference as it's down to the application.

Reply:
Very true, it will be the context that dictates which is 'better'. Regardless was worth emphasising the difference as it is I think quite subtle, and the re-write is very clear and complete. Nice touch with the suite.rc excerpts and diagrams!